Modules/Core/QuadEdgeMesh/wrapping/itkQuadEdgeMeshBase.wrap

```diff
@@ -19,6 +19,12 @@ foreach(d ${ITK_WRAP_IMAGE_DIMS})
 endforeach()
 itk_end_wrap_class()
 
+itk_wrap_class("itk::PointSetBase" POINTER)
+foreach(d ${ITK_WRAP_IMAGE_DIMS})
+  itk_wrap_template("MC${ITKM_IT}QEMPF${d}" "itk::MapContainer< ${ITKT_IT}, itk::QuadEdgeMeshPoint< float, ${d} > >")
+endforeach()
+itk_end_wrap_class()
+
```

Comment:
Even though this PR successfully passed the CI, it now looks like it made ITK.Windows.Python unhappy! It says at https://open.cdash.org/viewBuildError.php?type=1&buildid=9948358

```
itkQuadEdgeMeshBase: warning(4): ITK type not wrapped, or currently not known: itk::PointSetBase< itk::MapContainer< unsigned long long, itk::QuadEdgeMeshPoint< float, 2, itk::GeometricalQuadEdge< unsigned long long, unsigned long long, bool, bool > > > >
itkQuadEdgeMeshBase: warning(4): ITK type not wrapped, or currently not known: itk::MapContainer< unsigned long long, itk::QuadEdgeMeshPoint< float, 2 > >
itkQuadEdgeMeshBase: warning(4): ITK type not wrapped, or currently not known: itk::QuadEdgeMeshPoint< float, 2 >
```

Any idea?

Reply:
Isn't `ITKT_IT` specifier for `IdentifierType`, which should in turn be `unsigned long long` on MSVC?

Reply:
After a quick look, nothing jumps out as wrong. Best way to debug this is to set up a local Python build, and then iteratively tweak the wrap file and build.

Reply:
> Isn't `ITKT_IT` specifier for `IdentifierType`, which should in turn be `unsigned long long` on MSVC?

Thanks @dzenanz, looks like you're right 👍  I still wonder why `itk::IdentifierType` isn't just spelled out in the wrap files. "${ITKT_IT}" isn't very readable, is it?

Reply:
All of those `ITKT_IT`-style specifiers are abbreviated. The list is here: https://itk.org/ITKSoftwareGuide/html/Book1/ITKSoftwareGuide-Book1ch9.html#x55-1520009.5

See also a PR which fixes `ITKT_IT`: https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/pull/215

Reply:
Thanks @dzenanz I still find it hard to grasp all those `${ITKA_B}`...`${ITKY_Z}` abbreviations, but maybe it's just me  🤷

Reply:
I guess that `PointSetBase` should be wrapped with equivalent parameters to `QuadEdgeMeshTraits`. I guess it is not so right now?

Reply:
For every `PointSet` wrapping, there should be a `PointSetBase` wrapping. (But `PointSetBase` does not need as many wrappings as `PointSet`, because `PointSetBase` does not need to have a separate wrapping for each PixelType.)

Reply:
Should we also in this case use `${ITKT_F}` instead of `float`? But then, is `${ITKT_F}` anything more than just an "abbreviation" for `float`?

Reply:
@N-Dekker thanks for the follow-up.

Yes, this is likely related to the difference between the definition of `itk::IdentifierType` on Windows and other platforms. On Windows it resolves to `unsigned long long` to ensure 64-bit, on Unix `unsigned long`, which is already always 64-bit.